search/tasks.py

```diff
@@ -4,30 +4,52 @@
 
 from micromasters.celery import async
 from search.api import (
+    index_program_enrolled_users as _index_program_enrolled_users,
+    remove_program_enrolled_user as _remove_program_enrolled_user,
     index_users as _index_users,
     remove_user as _remove_user
 )
 
 
 @async.task
-def index_users(users):
+def remove_program_enrolled_user(user):
+    """
+    Remove program-enrolled user from index
+
+    Args:
+        user (User): A program-enrolled user to remove from index
+    """
+    _remove_program_enrolled_user(user)
+
+
+@async.task
+def index_program_enrolled_users(program_enrollments):
     """
     Index profiles
 
     Args:
-        users (iterable of User):
-            Iterable of Users
+        program_enrollments: Iterable of ProgramEnrollments
```

Comment:
Can you add a type here for `program_enrollments`? It should go in parentheses after the name according to the sphinx-napoleon convention: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings